app/src/main/java/com/teamagam/gimelgimel/app/map/actions/timeplay/DateTimePicker.java

```diff
@@ -66,4 +66,10 @@ public void showAlertErrorAndReopenPicker() {
         .create();
     errorAlert.show();
   }
+
+  public interface DateDisplayer {
```

Comment:
This interface doesn't belong here either.....
Again - interfaces should reside where they're used.
Why is that ? to enhance the code-locality - code parts that are related should reside in the same place.
Whether  it's by placing it in the same package or placing an inner-interface right next to where it's consumed.

Since this DateDisplayer interface is used solely by one class it can be an inner interface (meaning, it's inside another class). BUT, it then must reside in its using class.